documentation/tutorials/BlockchainScraper.md

```diff
@@ -63,18 +68,20 @@ For your Go code, also add an entry into the docker-compose file so that your Do
 An example entry should look something like this
 
 ```
-btc: 
-  build:
-    context: ../../../..
-    dockerfile: github.com/diadata-org/diadata/blockchain-scrapers/Dockerfile-btc
-  image: blockchain-scrapers_btc 
-  networks:
-    - scrapers-network
-  logging:
-    options:
-      max-size: "50m"
-  secrets:
-    - api_diadata
+  btc:
+    build:
+      context: ../../../..
+    build:
+      context: $GOPATH
+      dockerfile: $GOPATH/src/github.com/diadata-org/diadata/build/Dockerfile-btc
```

Comment:
2 build sections ?

Reply:
I copied that from `docker-compose.bitcoin.yml` currently that is the content on the file.
Should I update the doc to
``` Docker
    build:
      context: ../../../..
      context: $GOPATH
      dockerfile: $GOPATH/src/github.com/diadata-org/diadata/build/Dockerfile-btc
```

Reply:
Ah OK